### ISSUE TITLE
Use SNS or Twilio to send an SMS message

### ISSUE THREAD

**Issue by krp** [kris-classes/restart#9]

**Comment by bevanryken** [comment]
Will try Twilio first then SNS - Need to figure out if it is SNS from AWS or a different service

**Comment by bevanryken** [comment]
Had issues with registering phone number as well, page must have a bug. Goto your user settings on the top right corner, and update / re-enter from there to work around

**Comment by P00t1s** [comment]
Bev, any luck with your own phone number?  I've tried the two numbers I have access to, neither of them are getting sent the sms to add the number as a verified one

**Comment by bevanryken** [comment]
Sent a message to 2Degrees to see if they can do a special request from my Twilio sample number to my mobile and unfilter. Probably won't happen but worth a try. Also checked on Twilio if there were geo-tagging restrictions, had a look through the Twilio docs - So much you can do with this service.

Note: 
For results, in Python, I get a response such as SM0149bbf2db9915596c07e8e8402c7e79
 When checking logs under monitor, also says that it has successfully sent the message, so can only presume that it is prevented fro the Telco side.



![Image](https://user-images.githubusercontent.com/112356853/203282563-08d7171d-8c17-409e-a42b-b550c6bdd9ed.png)

**Comment by P00t1s** [comment]
Yeah, with this I'm going to guess that our messages are being sent from Twilio, and are being stopped by the telco provider considering the short code message that holds the 2fa verification code functions fine
